pom.xml

```diff
@@ -43,7 +43,7 @@
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
-      <version>23.0</version>
+      <version>23.5-jre</version>
```

Comment:
Can't we omit this version specification now that we're depending on parent 154?

Reply:
Actually, should we still be depending on `otj-parernt-oss` here?  Shouldn't we be using the "Spring" parent?

Reply:
I do agree we could do that, but let's defer re-parenting this to a separate change.